### PR TITLE
TM-5070 Pt1 - Renaming Bureau Exceptions functions and file names

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
These Dual PRs will just be for renaming. The heavier updates will come in forks of these PRs. I'll point out the the minimal non-renaming changes that are being updated in this PR, for awareness.

Dual Merge: 
- [API PR](https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/870)
- ~[Mock PR](https://github.com/MetaPhase-Consulting/mock-fsbid/pull/415)~ - already merged in

[TM-5070](https://metaphase.atlassian.net/browse/TM-5070)

---
Requested Changes will be addressed in forked PR

[TM-5070]: https://metaphase.atlassian.net/browse/TM-5070?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Constants/Menu.js [hunk 1]
```diff
@@ -269,7 +269,7 @@ export const GET_PROFILE_MENU = () => MenuConfig([
         route: '/profile/bureau/availablebidders',
         icon: 'users',
         roles: [
-          'super_user',
+          'superuser',
```
Comment (elizabeth-jimenez) [inline src/Constants/Menu.js:272]: something we noticed during the Release

File: src/sass/_bureauExceptions.scss [hunk 2]
```diff
@@ -43,6 +40,7 @@
     }
 }
 .bureau-exception-text-input {
+    color: red!important;
```
Comment (elizabeth-jimenez) [inline src/sass/_bureauExceptions.scss:43]: self reminder 🌸

**Comment by elizabeth-jimenez** [comment]
One thing I really wanted to do with all of this renaming is consolidating Bureau Exception, Bureau Exceptions, Bureau Exception List, etc all to Bureau Exceptions

File: src/actions/bureauExceptions.js [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+    })
+      .then(({ data }) => {
+        batch(() => {
+          dispatch(userBureauExceptionsAndMetaDataSuccess(data));
+          dispatch(userBureauExceptionsAndMetaDataErrored(false));
+          dispatch(userBureauExceptionsAndMetaDataLoading(false));
+        });
+      })
+      .catch((err) => {
+        if (err?.message === 'cancel') {
```
Comment (elizabeth-jimenez) [inline src/actions/bureauExceptions.js:148]: cancel token was set up, but it was never being listened for - that has been implemented in this PR

File: src/actions/bureauExceptions.js [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+}
+export function userBureauExceptionsAndMetaDataFetch(query = {}) {
+  return (dispatch) => {
+    if (cancelUserBureauExceptionsAndMetaData) { cancelUserBureauExceptionsAndMetaData('cancel'); dispatch(userBureauExceptionsAndMetaDataLoading(true)); }
+    batch(() => {
+      dispatch(userBureauExceptionsAndMetaDataLoading(true));
+      dispatch(userBureauExceptionsAndMetaDataErrored(false));
+    });
+    const q = convertQueryToString(query);
+    api().get(`/fsbid/bureau_exceptions/metadata/?${q}`, {
```
Comment (elizabeth-jimenez) [inline src/actions/bureauExceptions.js:137]: this was a get with a body, so was never passing back the necessary user data - that has been updated in this PR too

File: src/actions/bureauExceptions.js [hunk 5]
```diff
@@ -0,0 +1,209 @@
...
+        if (err?.message === 'cancel') {
+          batch(() => {
+            dispatch(bureauExceptionsLoading(true));
+            dispatch(bureauExceptionsErrored(false));
+          });
+        } else {
+          batch(() => {
+            /* eslint-disable no-console */
+            console.log('🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳 reminder: verify the structure is []');
+            console.log('🥳🥳🥳🥳🥳🥳🥳🥳🥳🥳 redux says so, so update there too if not');
```
Comment (M00staff) [inline src/actions/bureauExceptions.js:129]: typo?

File: src/actions/bureauExceptions.js [hunk 6]
```diff
@@ -0,0 +1,209 @@
...
+      })
+      .catch((err) => {
+        if (err?.message === 'cancel') {
+          batch(() => {
+            dispatch(userBureauExceptionsAndMetaDataLoading(true));
+            dispatch(userBureauExceptionsAndMetaDataErrored(false));
+          });
+        } else {
+          batch(() => {
+            // 📝📝 reminder dbl check expected data structure
```
Comment (M00staff) [inline src/actions/bureauExceptions.js:71]: dunno if you still need/want this to be here?

**Review by M00staff — APPROVED** [review]
didn't originally realize this was just to rename everything.

regardless, everything is wired up correctly, no typos between action calls > action names > action types & reducer names... ctrl & F'd every one of them 😂

one thing (i assume this is for another PR) but when i click this bureau checkbox locally 
image

i get a crash 

image

I'm assuming that's known and would be addressed later. 

This should be OK to merge in.

**Comment by elizabeth-jimenez** [comment]
@M00staff thanks for the review! I'll come back and address your comments on both PRs before I open up the other two PRs 🎊

**Pull request merged**